pkg/telemetryservice/sql/telemetryservicesql.go

```diff
@@ -12,6 +13,11 @@ import (
 	"github.com/edgenesis/shifu/pkg/telemetryservice/sql/tdengine"
 )
 
+const (
+	PasswordSecretField = "password"
```

Comment:
this const appeared again. shall we move to an more common place to make one definition cross the project?

Reply:
ok, telemetry imports these two from deviceshifubase.